### PR TITLE
Chore/ot11

### PR DESCRIPTION
**Pull request by KyleTryon**

### CODE REVIEW

**Comment by orb-publisher** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/bats?version=dev:d86ef20620b914e52fd730c2c26499bc4d720f6f

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,94 +1,37 @@
...
-parameters:
-  # These pipeline parameters are required by the "trigger-integration-tests-workflow"
-  # job, by default.
-  run-integration-tests:
-    type: boolean
-    default: false
-  dev-orb-version:
-    type: string
-    default: "dev:alpha"
+  orb-tools: circleci/orb-tools@11.1
```
Comment (Jaryt) [inline .circleci/config.yml:4]: We're on orb tools 11.5 now. Need to update the template sometime soon.

**Review by Jaryt — APPROVED** [review]
Looks good!

**Comment by Peter-Darton-i2** [comment]
While you're "in here" ... may I suggest that, at the end of the `scripts/install_bats.sh` script, instead of calling `runInstallBats` unconditionally, the code should be modified to do something like:
```
if ! which bats > /dev/null 2>&1; then
  runInstallBats
fi
```

This will then allow the installation script to "silently succeed" if it's run in an environment where bats is already installed (e.g. it's being run twice, or it's being used in a docker image that had bats installed already)

(if you prefer, I can raise a PR that does this)

**Comment by orb-publisher** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/bats?version=dev:3a862ec831e9660d3662a5526f5ad641fb91af06

**Comment by KyleTryon** [comment]
> While you're "in here" ... may I suggest that, at the end of the `scripts/install_bats.sh` script, instead of calling `runInstallBats` unconditionally, the code should be modified to do something like:
> 
> ```
> if ! which bats > /dev/null 2>&1; then
>   runInstallBats
> fi
> ```
> 
> This will then allow the installation script to "silently succeed" if it's run in an environment where bats is already installed (e.g. it's being run twice, or it's being used in a docker image that had bats installed already)
> 
> (if you prefer, I can raise a PR that does this)

Absolutely 👍 . Just going to break these changes into new PRs. We'll have a (most likely major) new version coming out shortly.

**Comment by orb-publisher** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/bats?version=dev:ff0affcd11d86a1617fe621785ba07dfc269f682

**Pull request merged**